elastic/kibana
# [ResponseOps][Alerting] Fix stackAlerts plugin missing rac API auth scope

## Summary

Adds the `['rac']` API access scope to the Stack Alerts feature to correctly authenticate alerts API endpoints with the `stackAlerts` permission.
Also adds a dedicated API integration test for the impacted endpoint and permission set.

## Release note

Fix Stack Alerts feature API access control

## To verify

1. Create rules that fire alerts in Stack management
2. Wait for alerts to be created
3. Create a role with only `Stack Management > Rules : Read` privilege
4. Create a user with that role
5. In another window, open Kibana with the newly created user
6. Check that the Stack Management > Alerts page renders correctly, not showing any missing 403 error toasts
